src/methods/wallet-send-calls.ts

```diff
@@ -21,29 +24,43 @@ import {
   validateParams,
 } from '../utils/validation';
 
+const CapabilitiesStruct = record(
+  string(),
+  type({
+    optional: optional(boolean()),
+  }),
+);
+
 const SendCallsStruct = tuple([
   object({
     version: nonempty(string()),
+    id: optional(StrictHexStruct),
     from: HexChecksumAddressStruct,
     chainId: optional(StrictHexStruct),
     calls: array(
       object({
         to: optional(HexChecksumAddressStruct),
         data: optional(StrictHexStruct),
         value: optional(StrictHexStruct),
+        capabilities: optional(CapabilitiesStruct),
       }),
     ),
-    capabilities: optional(type({})),
+    capabilities: optional(CapabilitiesStruct),
   }),
 ]);
 
 export type SendCallsParams = Infer<typeof SendCallsStruct>;
 export type SendCalls = SendCallsParams[0];
 
+export type SendCallsResult = {
+  id: Hex;
```

Comment:
should this be `StrictHexStruct`?

Reply:
This isn't a struct but a standard type as it's the result so no validation needed.